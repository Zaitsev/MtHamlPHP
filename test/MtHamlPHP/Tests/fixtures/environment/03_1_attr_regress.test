--FILE--
<?php
$env = new MtHamlPHP\Environment('php', array('enable_escaper' => false));
echo $env->compileString($parts['HAML'], "$file.haml");

--HAML--
:haml
    use_runtime :   false
:php
    $str='a/';
%a{:href=>"#{$str}&page=1"}

--EXPECT--
<?php
     $str='a/';
?>
<a href="<?php echo($str) ;?>&page=1"></a>

--EXECUTED--
<a href="a/&page=1"></a>

