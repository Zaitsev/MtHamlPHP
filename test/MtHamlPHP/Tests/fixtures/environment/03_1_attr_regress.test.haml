:haml
    use_runtime=>false
%a{:href=>"#{$str}&page=1"}
