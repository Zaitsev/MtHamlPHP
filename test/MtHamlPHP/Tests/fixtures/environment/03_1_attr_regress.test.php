<?php
$env = new MtHamlPHP\Environment('php', array('enable_escaper' => false));
echo $env->compileString($parts['HAML'], "$file.haml");
