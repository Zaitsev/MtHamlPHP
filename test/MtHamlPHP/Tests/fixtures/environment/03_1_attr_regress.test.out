<a href="<?php echo($str) ;?> &page=1"></a>
